pyproject.toml

```diff
@@ -40,6 +40,7 @@ default_section = 'THIRDPARTY'
 force_grid_wrap = 0
 include_trailing_comma = true
 known_first_party = 'datadog_checks'
+known_third_party = 'kafka,kubernetes,openstack,rethinkdb'
```

Comment:
Why is this necessary now? This concerns me.

Reply:
Somehow isort think that `kafka,kubernetes,openstack,rethinkdb` are `first_party` ... maybe a bug in isort 5

Reply:
Can you please file an issue?

Reply:
@ofek Yes done here: https://github.com/PyCQA/isort/issues/1467

Reply:
Fixed by https://github.com/DataDog/integrations-core/pull/7539/commits/828a2cac5751214e5ebc10ed4a1871e3f0a885ec

See also https://github.com/PyCQA/isort/issues/1467#issuecomment-689289138

Reply:
Should we wait for that to be released?

Reply:
@hithwen there is no code change from `isort`, `src_paths` is an existing config.